build.gradle

```diff
@@ -76,16 +76,35 @@ String getGithubToken() {
   }
 }
 
+// Explicitly adding dependency on "shadowJarExecutable" as Gradle does not it set via "releaseAssets" property
+tasks.named("githubRelease") {
+  dependsOn { project(":ktlint").tasks.named("shadowJarExecutable") }
+}
+
 githubRelease {
   token getGithubToken()
   owner "pinterest"
   repo "ktlint"
+  tagName project.properties['VERSION_NAME']
+  releaseName project.properties['VERSION_NAME']
+  releaseAssets project.files({
```

Comment:
there's also an `.asc` file and the sources zip - are they also being added?

Reply:
if it will be in `shadowJarExecutable` folder - should pick up. If not - easy to add manually here